resources/web/README.md

```diff
@@ -1,15 +1,85 @@
+display: flex;                                                                                                          
+flex-wrap: nowrap;                                                                                                      
+overflow-x: auto;                                                                                                                                                                                                                             
+}                                                                                                                       
+.scroll-child {                                                                                                             
+flex: 0 0 auto;                                                                                                        
+ margin-right: 36px;                                                                                                     
+font-size: 24px                                                                                                       
+}                                                                                                                     
+</style>                                                                                                                                 
+
+## What Web Development Tools Are There?
+
+### Top 10 Frontend Libraries
+
+<LevelWithButton :imageRight="false" desc="Javascript frameworks are the way-to-go with frontend development. Checkout this list on the top 10 frameworks to get started with UI" link="https://medium.com/javarevisited/10-of-the-most-popular-javascript-frameworks-libraries-for-web-development-in-2019-a2c8cea68094" image="https://miro.medium.com/max/3840/1*4cFhtuq6zRDqJ6p4s2pQ6g.jpeg" button="Check it out!" /> 
+
+### Web development tools you should know
+When you start developing your web application, you will need an array of tools to keep help your development agile and your code scalable. This includes using version control (like Git) and deployment containers (like Docker). 
+<LevelWithButton desc="Here are some essential tools you should definetly know before you start your next project!" link="https://www.freecodecamp.org/news/handy-web-development-toolkit/" image="https://www.freecodecamp.org/news/content/images/size/w2000/2020/04/screely-1586183781361.png" button="Check it out!" /> 
+
+## Selected frameworks to help you get started!
+
+### ReactJS
+
+<LevelWithButton desc="ReactJS is a frontend library created by facebook to help create modern and responsive UIs. It's one of the most well-known frameworks out there and is really easy to learn! Checkout this tutorial!" button="Get started with ReactJS!" link="https://reactjs.org/tutorial/tutorial.html" image="https://miro.medium.com/max/900/1*EntHChgUyirgbZ9A3zTxkA.png" />
```

Comment:
The Hacklab is going to have a react resources page as well under resources/react 

Let's link that page, and give an overview of React as the description.